tests/test_lib_user.py

```diff
@@ -8,14 +8,16 @@
 PWFILE2 = "tests/testdata/passwords"
 
 from .base import MyTestCase
-from privacyidea.lib.resolver import (save_resolver)
+from privacyidea.lib.resolver import (save_resolver, delete_resolver)
 from privacyidea.lib.realm import (set_realm, delete_realm)
 from privacyidea.lib.user import (User, create_user,
                                   get_username,
                                   get_user_info,
                                   get_user_list,
                                   split_user,
                                   get_user_from_param)
+import ldap3mock
+from test_lib_resolver import objectGUIDs, LDAPDirectory_small
```

Comment:
Maybe we should use a special file for these definitions so we don't have to import other test-scripts.